src/main/java/frc/robot/AutoCommands.java

```diff
@@ -65,15 +63,6 @@ public Command scoreSpeaker() {
         drivetrainSubsystem, shooterSubsystem, turretSubsystem, ledSubsystem, elevatorSubsystem::isParked);
   }
 
-  /**
-   * Command to intake a note into the turret
-   * @return new command
-   */
-  public Command intakeToTurret() {
-    return new IntakeToTurretCommand(intakeSubsystem, turretSubsystem, amperSubsystem)
-        .deadlineWith(new LEDAlternateCommand(ledSubsystem, NOTE_COLOR, Color.kBlue, Seconds.one()));
```

Comment:
This ran the intake command and made the LEDs flash. Don't we want to keep this?